addons/website_sale/views/templates.xml

```diff
@@ -1876,7 +1876,9 @@
                             <div t-if="product_image._name == 'product.image' and product_image.embed_code" class="d-flex align-items-center justify-content-center h-100 embed-responsive embed-responsive-16by9">
                                 <t t-raw="product_image.embed_code"/>
                             </div>
-                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image"}'/>
+                            <div t-else="" class="d-flex align-items-center justify-content-center h-100">
+                                <div t-field="product_image.image_1920" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image"}'/>
+                            </div>
```

Comment:
Let's not make that kind of changes in a stable version. Changing the XML structure is only to fix important bugs, this is not one ;) Move that commit in the master branch.

Reply:
The commit is now part of https://github.com/odoo/odoo/pull/66132